internal/storage/bucket_handle_test.go

```diff
@@ -364,6 +367,81 @@ func (t *BucketHandleTest) TestGetProjectValueWhenGcloudProjectionIsDefault() {
 	AssertEq(storage.ProjectionFull, proj)
 }
 
+func (t *BucketHandleTest) TestListMinObjectMethod(){
+
+    //creating dummy object for testing
+       content := "Test Object creation"
+       objName := "test_obj_6_attr"
+       contentType :=    "dummy content type"
+       contentLanguage := "dummy content language"
+       contentEncoding :=   "dummy content encoding"
```

Comment:
There are constants defined for this at the top of the file. why are you not using them.